packages/browser/src/sdk.ts

```diff
@@ -116,6 +117,31 @@ export function init(options: BrowserOptions = {}): void {
     options.sendClientReports = true;
   }
 
+  // inside Chrome extension (chrome.* API)
+  const windowWithMaybeChrome = WINDOW as typeof WINDOW & { chrome?: { runtime?: { id?: string } } };
+
+  // inside Firefox/Safari extension (browser.* API)
+  const windowWithMaybeBrowser = WINDOW as typeof WINDOW & { browser?: { runtime?: { id?: string } } };
+
+  if (
+    (windowWithMaybeBrowser &&
+      windowWithMaybeBrowser.browser &&
+      windowWithMaybeBrowser.browser.runtime &&
+      windowWithMaybeBrowser.browser.runtime.id) ||
+    (windowWithMaybeChrome &&
+      windowWithMaybeChrome.chrome &&
+      windowWithMaybeChrome.chrome.runtime &&
+      windowWithMaybeChrome.chrome.runtime.id)
+  ) {
+    consoleSandbox(() => {
+      // eslint-disable-next-line no-console
+      console.error(
+        '[Sentry] You cannot run Sentry this way in an extension, check: https://docs.sentry.io/platforms/javascript/troubleshooting/#setting-up-sentry-in-shared-environments-eg-browser-extensions',
```

Comment:
```suggestion
        '[Sentry] You cannot run Sentry this way in a browser extension, check: https://docs.sentry.io/platforms/javascript/troubleshooting/#setting-up-sentry-in-shared-environments-eg-browser-extensions',
```

maybe a bit clearer, wording wise? 🤔